habitat-baselines/habitat_baselines/common/baseline_registry.py

```diff
@@ -149,6 +149,23 @@ def register_storage(cls, to_register=None, *, name: Optional[str] = None):
     def get_storage(cls, name: str):
         return cls._get_impl("storage", name)
 
+    @classmethod
+    def register_agent_access_mgr(
+        cls, to_register=None, *, name: Optional[str] = None
```

Comment:
Could you describe what to_register means?

Reply:
Also, what is the use case of name being None? Is this for single agent trainer?